test/jdk/java/net/SocketPermission/SocketPermissionIm.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * @test
+ * @bug 8243376
+ * @summary SocketPermission implies(Permission p) spec allows, If the object was initialized with a single IP address and one of p's IP addresses is equal to this object's IP addr
+ * @run java -Dsun.net.inetaddr.ttl=0 SocketPermissionIm
+ */
+
+import java.net.SocketPermission;
+import java.net.InetAddress;
+import java.io.*;
+
+public class SocketPermissionIm {
+           public static void main(String[] args) throws Exception {
```

Comment:
nit: could you please fix the formatting in this file? 4 space indentation + 1 space after `do` and 1 space before and after `while`.